vector-im/element-android
# New App Layout: Adds New Chat Bottom Sheet

## Type of change


## Content



## Motivation and context

Closes https://github.com/vector-im/element-android/issues/6717

This is the new way we differentiate between a new DM or a new create room chat when using the FAB on the main screen

## Screenshots / GIFs

<img width="200" alt="image" src="https://user-images.githubusercontent.com/20701752/185228306-cdfe533f-cea5-4ded-9416-3d34a8ec1894.png">


## Tests



- Enable the New App Layout Feature Flag
- Click the primary FAB
- See bottom sheet
- Ensure both options work

## Tested devices

- OS version(s): Android 12

## Checklist
